db/migrate/20181113140945_create_unreceived_matches.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUnreceivedMatches < ActiveRecord::Migration[5.2]
+  def change
+    create_table :unreceived_matches do |t|
+      t.references :user, foreign_key: true
+      t.references :receiver, foreign_key: true
+      t.string :code
```

Comment:
We should probably use `null` constraints for every column. Also, we need a `received` boolean because gifts can later be delivered by the Santa, and our logistics folk need to register that.

Reply:
Done :+1: